packages/lesswrong/server/notificationTypesServer.tsx

```diff
@@ -383,3 +385,31 @@ export const NewRSVPNotification = serverRegisterNotificationType({
     </div>
   },
 });
+
+export const NewGroupOrganizerNotification = serverRegisterNotificationType({
+  name: "newGroupOrganizer",
+  canCombineEmails: false,
+  emailSubject: async ({ user, notifications }: {user: DbUser, notifications: DbNotification[]}) => {
+    const localGroup = await Localgroups.findOne(notifications[0].documentId)
+    if (!localGroup) throw new Error("Cannot find local group for which this notification is being sent")
+    return `You've been added as an organizer of ${localGroup.name}`;
+  },
+  emailBody: async ({ user, notifications }: {user: DbUser, notifications: DbNotification[]}) => {
+    const localGroup = await Localgroups.findOne(notifications[0].documentId)
+    if (!localGroup) throw new Error("Cannot find local group for which this notification is being sent")
+    
+    const groupLink = `${getSiteUrl().slice(0,-1)}/groups/${localGroup._id}`
+    
+    return <div>
+      <p>
+        Hi {user.displayName},
+      </p>
+      <p>
+        You've been assigned as a group organizer for <a href={groupLink}>{localGroup.name}</a> on {siteNameWithArticleSetting.get()}.
+      </p>
+      <p>
+        - {capitalize(siteNameWithArticleSetting.get())} Team
```

Comment:
I think you want to say The LessWrong Team here. siteNameWithArticleSetting is designed to allow you to switch between "on LessWrong" and "on the EA Forum". You can just use the regular site name and supply the article.